package.json

```diff
@@ -8,8 +8,10 @@
     "check-types": "tsc",
     "preview": "vite preview",
     "lint": "eslint --max-warnings 0 'src/**/*.ts' 'src/**/*.tsx' && knip",
+    "lighthouse": "SCRIVITO_ORIGIN=http://localhost:8080 PRERENDER_OBJ_ID=c2a0aab78be05a4e npm run prerender && npm run prerender-precompress && for run in $(seq 6); do lhci autorun && break; done",
     "prerender": "vite build && vite-node --options.deps.external='/scrivito/' src/prerenderContent.ts",
-    "prerender-preview": "vite preview --outDir dist-ssr",
+    "prerender-precompress": "precompress --include '(**).(css|html|js)' --types br dist-ssr",
+    "prerender-preview": "http-server dist-ssr --port 8080 --brotli -c31536000",
```

Comment:
This does not seem to work - at least on my machine.

Steps to reproduce:

```bash
SCRIVITO_ORIGIN=http://localhost:8080 npm run prerender && npm run prerender-preview
```

Visit http://localhost:8080 in a private browser window.

Expected: A redirect to /en is happening and the actual portal app is rendered.

Actual: The "index" page of the folder "en/" is visible:
<img width="1624" alt="Screenshot 2024-08-07 at 14 43 15" src="https://github.com/user-attachments/assets/5a4e3684-66fb-49ce-bc81-7480dd72989c">